services/github/src/main.js

```diff
@@ -186,12 +186,12 @@ const load = loader({
       return monitor.oneShot('syncInstallations', async () => {
         const gh = await github.getIntegrationGithub();
         const installations = (await gh.apps.getInstallations({})).data;
-        await Promise.map(installations, inst => {
+        await Promise.all(_.map(installations, inst => {
```

Comment:
There's no need for adding a dependency: `await Promise.all(installations.map(inst => { etc.` Please prune lodash from this PR

Reply:
Lodash was already being required in the files i changed. But yes i felt that too from chat on irc

Reply:
Yeah, wit's already used in a lot of places. I'm trying to get rid of it in tc-github, so I'd prefer we don't create new usages of lodash, especially in places like this. Here you don't even have to use any obscure ES7 trickery - just plain ES5 `map`